mate-panel/wayland-backend.c

```diff
@@ -71,6 +71,7 @@ wayland_panel_toplevel_update_placement (PanelToplevel* toplevel)
 		break;
 	case PANEL_ORIENTATION_BOTTOM:
 		anchor[GTK_LAYER_SHELL_EDGE_BOTTOM] = TRUE;
+		anchor[GTK_LAYER_SHELL_EDGE_RIGHT] = TRUE;
```

Comment:
This sound wrong, shouldn't this just follow the expand policy as done above? And if not for some reason, it feels like other orientations likely need similar treatment.

Reply:
Generally for legitimate non-obvious changes you probably should make it a bit clearer somewhere why that's done (commit message at least), and separate them from other changes (this ought not to be needed for a *build*, even if it was for a neat experience on Wayland)

Reply:
Re git/github: what you should do is create separate branches, and work on each thing separately.  For you WIP stuff, if you need the PR, just base your WIP branch on the PR branch, but don't push it.

To remove this change from this PR:
* modify the code to remove the change
* `git add mate-panel/wayland-backend.c` to record the modification
* `git commit --amend` to modify the topmost commit
* `git push -f origin wayland-only-buld` to publish the update to this PR (change "origin" if needed)

Then, create a new branch and add back your changes (just do `git checkout -b my-new-branch` and start hacking again).  If you need to modify this PR again, go back on this branch (`git switch wayland-only-buld`), hack on it, commit, push the new stuff for this PR.